tf2_geometry_msgs/test/test_tf2_geometry_msgs.cpp

```diff
@@ -123,21 +141,15 @@ TEST(TfGeometry, Conversions)
     geometry_msgs::msg::Transform tf_msg;
     tf2::convert(tf_, tf_msg);
 
-    EXPECT_NEAR(rotation.getX(), tf_msg.rotation.x, EPS);
-    EXPECT_NEAR(rotation.getY(), tf_msg.rotation.y, EPS);
-    EXPECT_NEAR(rotation.getZ(), tf_msg.rotation.z, EPS);
-    EXPECT_NEAR(rotation.getW(), tf_msg.rotation.w, EPS);
+    EXPECT_PRED3(CheckQuaternionNear, tf_msg.rotation, rotation, EPS);
```

Comment:
```suggestion
    // Calls function in first position, with arguments in second and third 
    // position along with epsilon value defined in the fourth position.
    EXPECT_PRED3(CheckQuaternionNear, tf_msg.rotation, rotation, EPS);
```

Add a note for the next person.  I had to look this one up.

Reply:
Disagree but committing here :)